tests/coverage-map/trivial.cov-map

```diff
@@ -0,0 +1,8 @@
+Function name: trivial::main
+Raw bytes (9): 0x[01, 01, 00, 01, 01, 03, 01, 00, 0d]
+Number of files: 1
+- file 0 => global file 1
+Number of expressions: 0
+Number of file 0 mappings: 1
+- Code(Counter(0)) at (prev + 3, 1) to (start + 0, 13)
```

Comment:
This seems to be line/col information. Please render it in a way similar to diagnostics.

Reply:
This is line/col information, but with a twist: LLVM coverage maps encode line numbers in relative form:
- A region's start line is relative to the previous region's start line
- A region's end line is relative to its own start line

These lines print the line numbers in their relative form, partly so that we can see what's actually being encoded, and partly because relative line numbers are less sensitive to inserted/deleted lines in the test file.

Reply:
So I can certainly change the rendering if desired, but I'd like it to reflect the fact that these aren't ordinary absolute line numbers in some way.

Reply:
I don't quite understand what you mean by “similar to diagnostics”. Do you mean something like this?

```
- Code(Counter(0)) at :(prev + 3):1 to :(start + 0):13
```

Reply:
yea kind of, but with the prev/start additions it doesn't seem too useful to reuse the colon syntax